### PR TITLE
SQL : select entity tags tests

### PR DESCRIPTION
**Pull request by unrealwork**
- [x]  Added tests.

### CODE REVIEW

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectEntityTagsTest.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    private static final String TEST_METRIC_NAME = TEST_PREFIX + "metric";
+    private static final String TEST_ENTITY1_NAME = TEST_PREFIX + "entity-1";
+    private static final String TEST_ENTITY2_NAME = TEST_PREFIX + "entity-2";
+    private static final String TEST_ENTITY3_NAME = TEST_PREFIX + "entity-3";
+
+
+    @BeforeClass
+    public static void prepareData() throws Exception {
+        List<Series> seriesList = Arrays.asList(
+                new Series() {{
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/select/SqlSelectEntityTagsTest.java]: registry

**Comment by Sild** [comment]
review done

**Comment by KutuzovGregory** [comment]
ok

**Pull request merged**